packages/toolshed/src/core/attestations.ts

```diff
@@ -0,0 +1,44 @@
+import { ethers, Wallet } from "ethers"
+
+import { IDisputeManager } from "@graphprotocol/subgraph-service"
+
+/**
+ * Creates an attestation data for a given request and response CIDs.
+ * @param disputeManager The dispute manager contract instance.
+ * @param signer The allocation ID that will be signing the attestation.
+ * @param requestCID The request CID.
+ * @param responseCID The response CID.
+ * @param subgraphDeploymentId The subgraph deployment ID.
+ * @returns The attestation data.
+ */
+export async function createAttestationData(
```

Comment:
lets use `generateAttestationData()` for consistency with the other helpers where we return a signed message